src/gpu/nvidia/sycl_cuda_helper.hpp

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+* Copyright 2020-2021 Intel Corporation
+* Copyright 2020 Codeplay Software Limited
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+#ifndef GPU_NVIDIA_SYCL_CUDA_HELPER_HPP
+#define GPU_NVIDIA_SYCL_CUDA_HELPER_HPP
```

Comment:
Nit: if name changes, please update this one, too.

Reply:
done